### PR TITLE
Support windows platform

### PR DESCRIPTION
**Pull request by lijy91**
VeryCapture_20210918151003

### CODE REVIEW

**Comment by carldressler** [comment]
Might want to also support the new Windows 11 context menu? This context menu looks a bit dated (fitting very well with Windows)

File: lib/src/method_channel.dart [hunk 1]
```diff
@@ -35,6 +37,7 @@ class ShowMenuArgs {
 
   Map<String, dynamic> toJson() {
     return {
+      'devicePixelRatio': window.devicePixelRatio,
```
[the remark this answers is not in the document]
Comment (lijy91) [reply lib/src/method_channel.dart:40]: I don’t know what logic is inside it, but I think this is the best way

[the remark this answers is not in the document]
Comment (lesnitsky) [reply lib/src/method_channel.dart:40]: lets move this logic to `lib/src/context_menu_region.dart` (devicePixelRatio is available from `MediaQuery.of(context)`

[the remark this answers is not in the document]
Comment (lijy91) [reply lib/src/method_channel.dart:40]: Why can't I use `window.devicePixelRatio` ?

[the remark this answers is not in the document]
Comment (lesnitsky) [reply lib/src/method_channel.dart:40]: depending on `'dart:ui`` in method channel doesn't seem like a good idea

[the remark this answers is not in the document]
Comment (lijy91) [reply lib/src/method_channel.dart]: I just removed the dependency of `dart:ui`

**Comment by lesnitsky** [comment]
Thanks! 👍

**Pull request merged**